src/zeit/solr/converter.py

```diff
@@ -274,6 +275,24 @@ def process(self, value, doc_node):
         self.append_to_node(unicode(type), doc_node)
 
 
+class VolumeIndex(Index):
+    # XXX kludgy special case to only apply to IVolume objects and not to other
+    # ICMSContent that has been adapted to find their IVolume.
+
+    def process(self, value, doc_node):
+        if not zeit.content.volume.interfaces.IVolume.providedBy(value):
+            return
+
+        solr_date = str(value.date_digital_published).replace(' ', 'T', 1)
+        solr_date = solr_date.replace('+00:00', 'Z')
+        self.solr = 'date_digital_published'
+        self.append_to_node(unicode(solr_date), doc_node)
+
+        if value.product:
```

Comment:
Wenn wir das Erscheinungsdatum zum Sortieren/Vorgänger/Nachfolger benutzen, sollten wir das auf IVolume dann vielleicht required machen? (Was immernoch nicht heißt, dass es vorhanden ist, in Punkto die ganzen alten 1950er Jahrgänge...)